test/ct/api/2-non-disruptive/test_components.tavern.yaml

```diff
@@ -763,6 +763,9 @@ stages:
                         type: str
                         enum:
                           - "X86"
+                          - "ARM"
```

Comment:
Good catch. This can be left as you've updated it, or even better would probably be to remove the enum completely since it's only there as the result of a bad copy/paste. No where else in the test requires a specific Arch other than cases that are verifying the Arch parameter specifically. That isn't the case here.